lib/identity_doc_auth/acuant/request.rb

```diff
@@ -1,5 +1,16 @@
 module IdentityDocAuth
   module Acuant
+    # https://documentation.help/AssureID-Connect/Error%20Codes.html
+    # 438 and 439 are frequent errors that we do not want to be notified of
+    IGNORED_ERROR_CODES = [438, 439]
+    class RequestError < StandardError
+      attr_reader :error_code
+      def initialize(message, error_code)
+        @error_code = error_code
+        super(message)
```

Comment:
Do we want to interpolate the error code into the message so we can make it easier to find in NewRelic? I don't think ivars get automatically reported?

```suggestion
      def initialize(message, error_code)
        @error_code = error_code
        super("#{error_code}: #{message}")
```

Reply:
Also WDYT of moving this RequestError class up a namespace so we can just use the same class in Acuant/LN?

Reply:
could refactor it, but the error code is in the message: https://github.com/18F/identity-doc-auth/blob/main/lib/identity_doc_auth/acuant/request.rb#L94-L98

Reply:
Agree on moving it up a namespace

Reply:
> could refactor it, but the error code is in the message: https://github.com/18F/identity-doc-auth/blob/main/lib/identity_doc_auth/acuant/request.rb#L94-L98

ah perfect, thanks!